lib/evm_database_ops.rb

```diff
@@ -132,6 +132,11 @@ def self.download(connect_opts, local_file)
     local_file
   end
 
+  def self.raise_failover_executed_event
```

Comment:
I'm thinking this probably belongs in `EvmDatabase` rather than `EvmDatabaseOps`

Ops seems to be used for more "meta" operations like backup, restore, start, and stop where `EvmDatabase` deals more with the data in the database (seeding, and other model/table specific operations).